Add Trait Activation docs (#5241)
segmentio/segment-docs: src/engage/trait-activation/trait-activation-setup.md

---
title: Trait Activation Setup
plan: engage-foundations
beta: true
---

> On this page, you'll learn how to set up the Destination that you'll use to get started with [Trait Enrichment](/docs/engage/trait-activation/trait-enrichment/) and [ID Sync](/docs/engage/trait-activation/id-sync/).  

Reviewer: ```suggestion
On this page, you'll learn how to set up the destination that you'll use to get started with [Trait Enrichment](/docs/engage/trait-activation/trait-enrichment/) and [ID Sync](/docs/engage/trait-activation/id-sync/).  
```

> info "Trait Activation in beta"
> Trait Activation is in beta, and Segment is actively working on this feature. Some functionality may change before it becomes generally available. [Contact Segment](https://segment.com/help/contact/){:target="_blank"} with any feedback or questions.

## Set up a Destination

> You'll first choose a Destinaton to connect to your Audience and use with Trait Activation. Trait Activation supports real-time and batch audiences for the Destinations listed in the table below. Select your Destination, view its Segment documentation, then follow the required setup steps. 

Reviewer: ```suggestion
You'll first choose a destination to connect to your audience and use with Trait Activation. Trait Activation supports real-time and batch audiences for the destinations listed in the following table. Select your destination, view its Segment documentation, then follow the required setup steps. 
```

| [Salesforce (Actions)](/docs/connections/destinations/catalog/actions-salesforce/) | Event          | 1. Connect Salesforce to your Segment space. <br> 2. Select **Actions** as the destination framework. <br> 3. From the **Settings** tab, authenticate with Salesforce using OAuth. <br> 4. Follow steps in the [Customizing Mappings documentation](/docs/connections/destinations/actions/#customizing-mappings). <br> 5. Enable the Destination and configured mappings.                  |
|[SendGrid Marketing Campaigns](/docs/connections/destinations/catalog/actions-sendgrid/)   |   Event           |  1. Connect SendGrid Marketing Campaigns to your Segment space. <br> 2. Enter your SendGrid Marketing Campaigns API key into the connection settings.                      |
|[LiveRamp](/docs/connections/destinations/catalog/actions-liveramp-audiences/)   |   Event           |  1. Set up your LiveRamp file drop. <br> 2. Connect LiveRamp Audiences to your Segment space.

## Resyncs

> Segment recommends creating a new Audience for Trait Enrichment and ID Sync. For existing Audience destinations, both Trait Enrichment and ID Sync won't resync the entire Audience. Only new data flowing into Segment will adhere to new trait settings. 

Reviewer: ```suggestion
Segment recommends creating a new audience for Trait Enrichment and ID Sync. For existing audience destinations, both Trait Enrichment and ID Sync won't resync the entire audience. Only new data flowing into Segment will adhere to new trait settings. 
```

[Contact Segment support](https://segment.com/help/contact/){:target="_blank"} if you'd like your Audience resynced with Trait Enrichment and ID Sync. 

> warning ""

> > For Audiences larger than 50 million users, it may take several hours, or even days, to sync. Only one resync is allowed at a time for each workspace. 

Reviewer: ```suggestion
> Audiences larger than 50 million users may take several hours, or even days, to sync. Only one resync is allowed at a time for each workspace. 
```
